doc/tutorial/text_analytics/solutions/generate_skeletons.py

```diff
@@ -27,7 +27,7 @@
 
     for line in input_file:
         linestrip = line.strip()
-        if len(linestrip) == 0:
+        if not linestrip:
```

Comment:
same, not equivalent

Reply:
As far as I know, they are equivalent. How couldn't they be?